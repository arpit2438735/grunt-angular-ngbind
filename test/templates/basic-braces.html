<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title></title>
</head>
<body>
    <div ng-app="App">
       <div class="app">{{someValue}}</div>
       <span>{{::spanBindValue}}</span>
    </div>
</body>
</html>