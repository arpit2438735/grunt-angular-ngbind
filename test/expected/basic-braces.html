<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title></title>
</head>
<body>
    <div ng-app="App">
       <div class="app" ng-bind="someValue"></div>
    </div>
</body>
</html>